com.muezzin.app.yml

```diff
@@ -0,0 +1,73 @@
+app-id: com.muezzin.app
```

Comment:
this app-id doesn't follow the requirements.

1. the domain `muezzin.com` doesn't exist, so it's not in the control of the developers
2. `app` isn't an accetable application name.

See https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
Okay I'll change it to io.github.muezzin ; is that good ?

Reply:
Oh didn't see this, I'll update it again.
Thanks!